examples/common/asset/DigitsFont.c

```diff
@@ -3,7 +3,7 @@
 /* Re-sized : True */
 /* Rotated : 0.0 deg */
 
-
+#include "../controls/__common.h"
```

Comment:
Why adding this?

Reply:
I am not sure why this was necessary since the modified file does not seem to use any of the symbols in the included header file.

In any case this has now been removed since the project builds fine without it.

Thanks